public/js/main.js

```diff
@@ -49,12 +49,13 @@ if (isFirefoxPanel()) {
   const prettyPrint = require("./utils/pretty-print");
 
   module.exports = {
-    bootstrap: ({ threadClient, tabTarget }) => {
+    bootstrap: ({ threadClient, tabTarget, toolbox }) => {
       // TODO (jlast) remove when the panel has L10N
       if (!window.L10N) {
         window.L10N = require("../../packages/devtools-local-toolbox/public/js/utils/L10N");
         L10N.setBundle(require("./strings.json"));
       }
+      window.toolbox = toolbox;
```

Comment:
this is temporary to support the contextmenu in the panel... i'll clean this up.